samcli/local/docker/lambda_image.py

```diff
@@ -31,7 +31,7 @@
 
 RAPID_IMAGE_TAG_PREFIX = "rapid"
 
-TEST_RUNTIMES = ["ruby3.3"]
+TEST_RUNTIMES = ["N/A"]
```

Comment:
I see that you tried to do `TEST_RUNTIMES = []` first (which seems like the most logical option) but it failed to build. 

However, the failure [in the build](https://github.com/aws/aws-sam-cli/actions/runs/12960487479/job/36154409106) is 
```
samcli/local/docker/lambda_image.py:34: error: Need type annotation for "TEST_RUNTIMES" (hint: "TEST_RUNTIMES: list[<type>] = ...")  [var-annotated]
```

So I feel it would be better to actually leave it empty and with that type annotation, which I think it should be something like this:

```
TEST_RUNTIMES: list[str] = []
```